mods/ts/weapons/energyweapons.yaml

```diff
@@ -39,17 +77,43 @@ MechRailgun:
 		DamageTypes: Prone100Percent, TriggerProne, FireDeath
 
 SonicZap:
-	ReloadDelay: 120
+	ReloadDelay: 180
 	Range: 6c0
 	Charges: yes
 	Report: SONIC4.AUD
-	Projectile: LaserZap
-		Color: 200,0,255,255
-		BeamWidth: 12
-		BeamDuration: 50
+	Projectile: LineImpactProjectile
+		Speed: 0c64
+		TicksBetweenImpacts: 5 # Roughly 18 impacts.
+		BeyondTargetRange: 0c256
+		MaxTargetRange: 50c0
+		MinRangeSource: 0c0
+		FalloffMaxReduction: 0
+		FalloffStartDistance: 49c0
+		FalloffEndDistance: 50c0
+		BeamWidth: 0c256
+		Blockable: true
+		BeamRenderWidth: 384
+		BeamFireDuration: 90
+		Color: 200,0,40,255
```

Comment:
This should be translucent (reduce the alpha from 255 to something lower), and at least twice (probably three times) the width.